app/services/encrypted_key_maker.rb

```diff
@@ -7,7 +7,8 @@ class EncryptedKeyMaker
   # @return [UserAccessKey]
   #
   def make(user_access_key)
-    raise 'user_access_key must be a UserAccessKey' unless user_access_key.is_a? UserAccessKey
+    raise "user_access_key must be a UserAccessKey - you passed #{user_access_key}" unless
+      user_access_key.is_a? UserAccessKey
```

Comment:
same thing --- really weird for me to see the condition for the statement bumped to the next line